apps/openmw/mwmechanics/summoning.hpp

```diff
@@ -22,7 +22,7 @@ namespace MWMechanics
                             float magnitude, float remainingTime = -1, float totalTime = -1);
 
         /// To call after all effect sources have been visited
-        void process();
+        void process(bool cleanup);
```

Comment:
Ok, I will add it, if my last commit is OK, and I will not have to looking for another implementation.